src/shared/components/query/CancerStudyTreeData.ts

```diff
@@ -151,9 +163,17 @@ export default class CancerStudyTreeData {
                     );
                 }
 
-                let searchTerms = isCancerType
-                    ? node.name
-                    : [node.name, (node as CancerStudy).description].join('\n');
+                const foundStudyTags =
+                    allStudyTags.length &&
+                    isStudy(node) &&
+                    allStudyTags.find(

```

Comment:
the naming of things here is kinda confusing. ```find``` will only ever return a single item, so should the variable not be singular?  but i guess then it has a ```tags``` collection?  so a single study can have multiple tags?

Reply:
Renamed in [2009473](https://github.com/cBioPortal/cbioportal-frontend/pull/4571/commits/2009473b6d81c094ba4246ee45947fd9d316e4fe)